aspnetcore/mvc/models/file-uploads/samples/3.x/SampleApp/SampleApp.csproj

```diff
@@ -1,11 +1,11 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net9.0</TargetFramework>
```

Comment:
This mostly just made debugging easier.  It's possible my changes work in 3.1 as well.